download/index.md

```diff
@@ -81,45 +82,51 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
-* [macOS arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.monterey.arm64.zip)
-* [Ubuntu xenial x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.xenial.x86_64.tar.gz)
-* [Ubuntu bionic x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.bionic.x86_64.tar.gz)
-* [Ubuntu bionic arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.bionic.arm64.tar.gz)
-* [Ubuntu focal x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.focal.x86_64.tar.gz)
-* [Ubuntu Snap x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.x86_64.snap)
-* [AppImage x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3-x86_64.AppImage)
-* [Redhat/CentOS 7 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.rhel7.x86_64.tar.gz)
-* [Redhat/CentOS 8 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.rhel8.x86_64.tar.gz)
-* [Debian bullseye x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.4-v3.bullseye.x86_64.tar.gz)
+### Redis Stack 6.2.6-RC1
+
+* [macOS x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.catalina.x86_64.zip)
+* [macOS arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.monterey.arm64.zip)
+* [Ubuntu xenial x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.xenial.x86_64.tar.gz)
+* [Ubuntu bionic x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.bionic.x86_64.tar.gz)
+* [Ubuntu bionic arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.bionic.arm64.tar.gz)
+* [Ubuntu focal x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.focal.x86_64.tar.gz)
+* [Ubuntu focal arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.focal.arm64.tar.gz)
+* [Ubuntu Snap x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.x86_64.snap)
+* [AppImage x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1-x86_64.AppImage)
+* [Redhat/CentOS 7 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.rhel7.x86_64.tar.gz)
+* [Redhat/CentOS 8 x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.rhel8.x86_64.tar.gz)
+* [Debian bullseye x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.bullseye.x86_64.tar.gz)
+* [Ubuntu Jammy x86_64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.jammy.x86_64.zip)
+* [Ubuntu Jammy arm64](https://packages.redis.io/redis-stack/redis-stack-server-6.2.6-RC1.jammy.arm64.zip)
 * [redis-stack](https://hub.docker.com/r/redis/redis-stack) and [redis-stack-server](https://hub.docker.com/r/redis/redis-stack-server) x86_64 and arm64 dockers are available on [Dockerhub](https://hub.docker.com/u/redis)
```

Comment:
This should probably say redis-stack docker?

Reply:
I guess it should be clear for users, because we say redis-stack and redis-stack-server dockers are available on Dockerhub